### PR TITLE
Dogsmind i miss

### PR DESCRIPTION
**Pull request by seanyboy49**
- [x] Dog's mind section

- [x] I miss new york section

- [x] DJ ban section

### CODE REVIEW

File: src/issues/issue-2/dj-ban.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+          fluid {
+            ...GatsbyImageSharpFluid_withWebp_tracedSVG
+          }
+        }
+      }
+    }
+  `)
+
+  return (
+    <Img style={{ zIndex: "1000" }} fluid={data.file.childImageSharp.fluid} />
```
Comment (seanyboy49) [inline src/issues/issue-2/dj-ban.js:19]: Needs to have high z index in order to cover the `New York` text from section above

File: src/issues/issue-2/gate-of-paradise.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { useStaticQuery, graphql } from "gatsby"
 import styled from "styled-components"
 
 import { findFileByName } from "../../utils"
-import { mediaQueries, breakpoints } from "../../styles/layout"
```
Comment (seanyboy49) [inline src/issues/issue-2/gate-of-paradise.js:7]: this wasn't being used

File: src/issues/issue-2/gate-of-paradise.js [hunk 3]
```diff
@@ -63,10 +63,6 @@ const ArticleWrapper = styled.div`
     flex-direction: column-reverse;
   }
 `
-const ImageWrapper = styled.div`
```
Comment (seanyboy49) [inline src/issues/issue-2/gate-of-paradise.js:66]: neither was this

File: src/issues/issue-2/dogs-mind.js [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+  }
+`
+const Title = styled.h1`
+  font-family: "HelveticaNeue-Bold", "Helvetica Neue Bold", "Helvetica Neue";
+  font-size: 19vw;
+  margin: 0 0 0 -1vw;
+`
+
+const ByLine = styled.h3`
+  font-family: "HelveticaNeue-Bold", "Helvetica Neue Bold", "Helvetica Neue";
```
Comment (JinaZhu) [inline src/issues/issue-2/dogs-mind.js:30]: is this the default font-family?

**Review by JinaZhu — APPROVED** [review]

**Pull request merged**